ethexe/consensus/src/validator/core.rs

```diff
@@ -147,14 +137,39 @@ impl ValidatorCore {
     // TODO #4741
     pub async fn aggregate_validators_commitment(
         &mut self,
-        _block: &SimpleBlockData,
+        block: &SimpleBlockData,
     ) -> Result<Option<ValidatorsCommitment>> {
-        // self.middleware.make_election_at(ElectionRequest {
-        //     at_block_hash: todo!(),
-        //     at_timestamp: todo!(),
-        //     max_validators: todo!(),
-        // });
-        Ok(None)
+        let SimpleBlockData { hash, header } = block;
+        let timelines = self
+            .db
+            .protocol_timelines()
+            .ok_or(anyhow!("protocol timelines not found"))?;
+
+        let block_era = era_from_ts(header.timestamp, timelines.genesis_ts, timelines.era);
+        let end_of_era = end_of_era_timestamp(block_era, timelines.genesis_ts, timelines.era);
+        let election_ts = end_of_era - timelines.election;
+
+        if header.timestamp < election_ts {
+            tracing::info!(
```

Comment:
is this is really should be "info!" ? it will be written for each block. I think this is more closer to debug information